index.html

```diff
@@ -193,14 +193,13 @@
     <h2><dfn>Performance Timeline</dfn></h2>
     <p>Each <a>ECMAScript global environment</a> has:</p>
     <ul>
-      <li>a <dfn>performance observer task queued flag</dfn> and an associated
-      list of <a>registered performance observer</a> objects which is initially
-      empty.
-      </li>
+      <li>a <dfn>performance observer task queued flag</dfn></li>
+      <li>a <dfn>list of <a>registered performance observer</a> objects</dfn> that is initially empty</li>
       <li>a <dfn>performance entry buffer</dfn> to store
-      <a>PerformanceEntry</a> objects. It is initially empty.
+      <a>PerformanceEntry</a> objects that is initially empty
       </li>
     </ul>
+    <p>The <dfn>relevant performance entry buffer</dfn> is the <a>performance entry buffer</a> associated with the <a>relevant global object</a>.</p>
```

Comment:
So this has nothing to do with the "each ECMAScript global environment" bit right before it, right?

"relevant global object" is a broken link in the rendered version of the spec...  I _assume_ you meant https://html.spec.whatwg.org/multipage/#concept-relevant-global but note that this needs as input the specific platform object you're talking about.  This comment about "relevant global object" applies to other places in this PR too..

Reply:
> So this has nothing to do with the "each ECMAScript global environment" bit right before it, right?

Correct. This definition is separate from the list defined above.

> I assume you meant https://html.spec.whatwg.org/multipage/#concept-relevant-global but note that this needs as input the specific platform object you're talking about. 

Great catch, thank you. Fixed.